packages/wonder-blocks-button/components/button-core.js

```diff
@@ -5,8 +5,8 @@ import {StyleSheet} from "aphrodite";
 import {LabelLarge} from "wonder-blocks-typography";
 import Color, {mix, fade} from "wonder-blocks-color";
 import {addStyle} from "wonder-blocks-core";
+import type {Handlers} from "wonder-blocks-core";
```

Comment:
Hmm - I feel like this name isn't descriptive enough. Maybe `ClickableHandlers` instead?